apps/pi-drone-detector/src/alert_handlers.py

```
@@ -0,0 +1,413 @@
+                'event': 'drone_alerts_batch',
+                'timestamp': datetime.now().isoformat(),
+                'alerts': self._pending_batch,
+            }
+
+            data = json.dumps(batch_data).encode('utf-8')
+
+            req = urllib.request.Request(
+                self._webhook_url,
+                data=data,
+                headers={'Content-Type': 'application/json'},
+            )
```

Ruff (0.14.10)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)